### ISSUE TITLE
Error when using alpha mating

### ISSUE THREAD

**Issue by a-tonchev** [danielgatis/rembg#217]
When I try to use alpha mating on a image it throws following error message:

![image](https://user-images.githubusercontent.com/54528540/163794621-f8c9dab3-e7b3-47fd-b6c5-28e497281ede.png)

According documentation it should work well.

Installation over pip3:
Python 3.8.10
pip3 20.0.2
ubuntu 20.04

**Comment by danielgatis** [comment]
can you attach the input image plz? so I will try to reproduce it here. thanks.

**Comment by a-tonchev** [comment]
Hi @danielgatis thanks for the fast response. This would be my image:

![input](https://user-images.githubusercontent.com/54528540/163816432-5361403c-7b5a-4ab8-91a3-24f19564f852.png)

**Comment by danielgatis** [comment]
@a-tonchev I think this issue is related to your enviroment.

Here it works.

rembg, version 2.0.12
Python 3.9.12
pip 22.0.4
ProductName:    macOS
ProductVersion: 12.3
BuildVersion:   21E230

```
rembg i ~/Desktop/unnamed.png ~/Desktop/output.png -a -ae 15
```
![output](https://user-images.githubusercontent.com/7380/163817099-132f147b-a274-4fc3-b513-e8747a65c800.png)

**Comment by a-tonchev** [comment]
@danielgatis I tryed it in my local ubuntu and also on remote ubuntu server on Hetzner, they both show the same error messages. Somehow the command just throws errors :(

Do you have any access to linux?

The commend does not work anywhere for me:

```
curl -s https://user-images.githubusercontent.com/54528540/163816432-5361403c-7b5a-4ab8-91a3-24f19564f852.png | rembg i > output.png -a -ae 15
```

I tryed also with python3.9 and pip 22 but it won't work

**Comment by a-tonchev** [comment]
Just tested it in onworks virtual Ubuntu, it just don't work:

![image](https://user-images.githubusercontent.com/54528540/163826020-4dabdf33-b245-4ec7-842d-a7ce074a1536.png)

**Comment by danielgatis** [comment]
@a-tonchev fixed! try install v2.0.13

**Comment by a-tonchev** [comment]
Hi @danielgatis it works now. Thanks, you got a Coffee from me ;)
